modules/check_in/app/services/v2/chip/client.rb

```diff
@@ -156,6 +156,28 @@ def refresh_precheckin(token:)
         end
       end
 
+      ##
+      # HTTP DELETE call to the CHIP API to delete check-in/pre check-in data
+      #
+      # @param token [String] CHIP token to call the endpoint
+      #
+      # @return [Faraday::Response]
+      #
+      def delete(token:)
+        connection.delete("/#{base_path}/actions/deleteFromLorota/#{check_in_session.uuid}") do |req|
+          req.headers = default_headers.merge('Authorization' => "Bearer #{token}")
+        end
+      rescue => e
+        log_exception_to_sentry(e,
```

Comment:
Based on more recent understanding (see https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/check_in/app/services/v2/chip/client.rb#L149), should this be recorded as an exception in Sentry? my thinking is that since it's a downstream error, not exactly an exception with vets-api code, it can be recorded as a 'message' instead?

Reply:
Recording as an exception would help us know the downstream server issue & notify them instead of relying on the downstream server dashboard.  But again, we can always get notified on the message level info.

Also, while taking a closer look at the [code](https://github.com/department-of-veterans-affairs/vets-api/blob/39199_Delete_endpoint_integration_with_retry_limit_in_redis/lib/sentry_logging.rb#L6-L15), calling `log_message_to_sentry` with `error` level seem to be same as `log_exception_to_sentry ` except the message getting logged.  Pls let me if that is not the case.

Reply:
It's different, `Raven.capture_message(message, level: level)` vs `Raven.capture_exception(exception.cause.presence || exception, level: level)`

Reply:
I think downstream errors that we're handling ourselves probably shouldn't be sent to Sentry as an exception, but I know we have done that in other parts of code as well in the past so if you feel strongly about it, I'd defer to your expertise.

Reply:
Sure Gaurav, I think we can handle this as a separate action.  We can look at the ways of getting notified from sentry or datadog and make changes in all parts.